src/LightInject/LightInject/LightInject.cs

```diff
@@ -4031,7 +4047,7 @@ private Action<IEmitter> CreateServiceEmitterBasedOnFactoryRule(FactoryRule rule
                 ServiceType = serviceType,
                 ServiceName = serviceName,
                 FactoryExpression = rule.Factory,
-                Lifetime = CloneLifeTime(rule.LifeTime)
+                Lifetime = CloneLifeTime(rule.LifeTime) ?? DefaultLifetime
```

Comment:
Not sure if DefaultLifetime makes sense here, but I figure it can't hurt, right?